pkgs/applications/misc/ayatana-indicators/default.nix

```diff
@@ -0,0 +1,23 @@
+{ generateSplicesForMkScope
+, makeScopeWithSplicing
+, lomiri
+}:
+
+let
+  keep = _self: { };
+  extra = _spliced0: { };
+in
+makeScopeWithSplicing
+  (generateSplicesForMkScope "ayatana-indicators")
+  keep
+  extra
+  (self:
+    let
+      inherit (self) callPackage;
+    in
+    {
+      ayatana-indicator-messages = callPackage ./ayatana-indicator-messages {
+        inherit (lomiri) cmake-extras;
+      };
+    }
+  )
```

Comment:
what are you using this for?

Reply:
Originally my idea was to have different package scopes for "vanilla" and "lomiri-enhancements" indicators, and there was a bad package test along with them in here. But Lomiri scope would need Lomiri enhancements in `libayatana-common`, which needs unmerged stuff. And if Lomiri stuff for non-Lomiri is the way I'll go with this, then this will be wholly obsolete & `passthru.ayatana-indicators` will be good enough for making them stand out.

Will rethink & retest along with the rest. This extra scope will prolly be dropped though, yeah.